src/components/CardListBox/CardFolderList/CardFolderList.tsx

```diff
@@ -0,0 +1,33 @@
+import Folder from "./Folder/Folder";
+import styles from "./CardFolderList.module.scss";
+import { UserFolderProps } from "@/constants/index.types";
+
+interface CardFolderListProps {
+  folders?: UserFolderProps[];
+  handleFolderClick: (folder: UserFolderProps) => void;
+  handleTotalBtnClick: () => void;
+}
+
+export default function CardFolderList({
+  folders,
+  handleFolderClick,
+  handleTotalBtnClick,
+}: CardFolderListProps) {
+  return (
+    <section className={styles.CardFolderList}>
+      <button className={styles.totalBtn} onClick={() => handleTotalBtnClick()}>
```

Comment:
이렇게 표현해주는것과 동일합니다~
```suggestion
      <button className={styles.totalBtn} onClick={handleTotalBtnClick}>
```

Reply:
html button의 type attribute는 기본값이 submit이기 때문에 button으로 명시적으로 잡아주면 좋을 것 같아요~
```suggestion
      <button type='button' className={styles.totalBtn} onClick={() => handleTotalBtnClick()}>
```